providers/statsig/src/main/java/dev/openfeature/contrib/providers/statsig/ContextTransformer.java

```
@@ -0,0 +1,65 @@
+package dev.openfeature.contrib.providers.statsig;
+
+import com.statsig.sdk.StatsigUser;
+import dev.openfeature.sdk.EvaluationContext;
+import dev.openfeature.sdk.Structure;
+import dev.openfeature.sdk.Value;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Transformer from OpenFeature context to statsig User.
+ */
+class ContextTransformer {
+    public static final String CONTEXT_APP_VERSION = "appVersion";
+    public static final String CONTEXT_COUNTRY = "country";
+    public static final String CONTEXT_EMAIL = "email";
+    public static final String CONTEXT_IP = "ip";
+    public static final String CONTEXT_LOCALE = "locale";
+    public static final String CONTEXT_USER_AGENT = "userAgent";
+    public static final String CONTEXT_PRIVATE_ATTRIBUTES = "privateAttributes";
+
+    static StatsigUser transform(EvaluationContext ctx) {
+        StatsigUser user = new StatsigUser(ctx.getTargetingKey());
```

Is it OK if the arg to `new StatsigUser()` here is null? If not you might want to throw invalid context.

---

done